### PR TITLE
[zh-doc] Localized getting-started/architecture.md

### PR DESCRIPTION
**Pull request by windsonsea**
Signed-off-by: Michael <haifeng.yao@daocloud.io>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
The website has a language switch of en/zh, but almost all pages are not localized.

**Which issue(s) this PR fixes**:
Have a try to add Chinese text.

**Special notes for your reviewer**:
https://karmada.io/zh/docs/getting-started/architecture/

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by RainbowMango** [comment]
We are going to do the translation work soon.
I'll take a look at it this week. 
cc @pigletfly

**Comment by windsonsea** [comment]
> We are going to do the translation work soon. I'll take a look at it this week. cc @pigletfly

If possible, I would like to join your localization team :)

**Comment by RainbowMango** [comment]
> If possible, I would like to join your localization team :)

That's great! Welcome!

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 
 - Karmada API Server
 - Karmada Controller Manager
 - Karmada Scheduler
 
-ETCD stores the karmada API objects, the API Server is the REST endpoint all other components talk to, and the Karmada Controller Manager perform operations based on the API objects you create through the API server.
+ETCD 存储 Karmada API 对象，API Server 是与所有其他组件通信的 REST 端点，Karmada Controller Manager 根据 API 服务器创建的 API 对象执行操作。
 
-The Karmada Controller Manager runs the various controllers,  the controllers watch karmada objects and then talk to the underlying clusters' API servers to create regular Kubernetes resources.
+Karmada Controller Manager 运行各种控制器，这些控制器监测 Karmada 对象，然后与底层集群的 API 服务器对话，以创建常规的 Kubernetes 资源。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md:17]: 这些控制器监测 Karmada 对象，然后与底层集群的API服务器对话 seems strange.

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 
-1. Cluster Controller: attach kubernetes clusters to Karmada for managing the lifecycle of the clusters by creating cluster object.
+1. Cluster Controller：将 Kubernetes 集群挂接到 Karmada，通过创建集群对象来管理集群的生命周期。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md:19]: 挂接 weired

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
 
-2. Policy Controller: the controller watches PropagationPolicy objects. When PropagationPolicy object is added, it selects a group of resources matching the resourceSelector and create ResourceBinding with each single resource object.
-3. Binding Controller: the controller watches ResourceBinding object and create Work object corresponding to each cluster with single resource manifest.
-4. Execution Controller: the controller watches Work objects.When Work objects are created, it will distribute the resources to member clusters.
+2. Policy Controller：该控制器监测 PropagationPolicy 对象。添加 PropagationPolicy 对象时，它会选择一组与 resourceSelector 匹配的资源，并用每个单独的资源对象创建 ResourceBinding。
 
+3. Binding Controller：该控制器监测 ResourceBinding 对象，并用单个资源清单创建每个集群对应的 Work 对象。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md:23]: 并用单个资源清单创建每个集群对应的 Work 对象 weird

**Review by Poor12 — COMMENTED** [review]
Seems to be mostly machine flipped. The expressions of some words are not very friendly to Chinese readers.

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md (hunk 3 above)
Comment (windsonsea) [reply i18n/zh/docusaurus-plugin-content-docs/current/getting-started/architecture.md]: > 并用单个资源清单创建每个集群对应的 Work 对象 weird

Modified, thanks.

**Comment by windsonsea** [comment]
Seems the en version has changed much. This can be closed. Thanks.

**Pull request closed without merging**